tests/utils/io/test_mutations_report.py

```diff
@@ -373,13 +373,13 @@ def test_filtered_mutation_creation_annovar(self):
             self.assertEqual(len(result), 13)
             self.assertEqual(result[0].rstrip(), "sample1	NC_000002.11	29445271	29445271	G	A	1-hotspot	620	359	4")
```

Comment:
I think this could also be simplified by using files and comparing them against each other. Here an example: https://github.com/hydra-genetics/cnv_sv/blob/develop/workflow/scripts/generate_pindel_config_test.py